common/persistence/client/quotas.go

```diff
@@ -30,23 +30,57 @@ import (
 )
 
 var (
-	CallerTypePriority = map[string]int{
-		headers.CallerTypeAPI:        0,
-		headers.CallerTypeBackground: 1,
+	CallerTypeDefaultPriority = map[string]int{
+		headers.CallerTypeAPI:        1,
+		headers.CallerTypeBackground: 2,
 	}
 
-	APIPriorityOverride = map[string]int{
+	APITypeCallInitiationPriorityOverride = map[string]int{
+		"StartWorkflowExecution":           0,
+		"SignalWithStartWorkflowExecution": 0,
+		"SignalWorkflowExecution":          0,
+		"RequestCancelWorkflowExecution":   0,
+		"TerminateWorkflowExecution":       0,
+		"GetWorkflowExecutionHistory":      0,
+		"UpdateWorkflow":                   0,
```

Comment:
I understand the top 3 ones, but why special treat these 4?

Reply:
This is copied from the frontend api priority definition, which I guess there were some discussion when that code is written?
https://github.com/temporalio/temporal/blob/master/service/frontend/configs/quotas.go#L41

I agree that the top 3 are the most important ones. Let me know your thoughts if the rest 4 should be removed.